stories/Components/Forms/Dropdowns/Multiselect/multiselect.scss

```diff
@@ -196,6 +218,11 @@
           right: inherit;
         }
       }
+      ul {
+        ul {
+          padding: 0 40px 0 0;
```

Comment:
@Ashish-srij Use spacing variable if available else use `rem` instead of `px`.

Reply:
@shishirphadke Done.